docs/API.md

```diff
@@ -275,6 +277,79 @@ MyStamp().clone().clone().clone().x === 42; // true
 ```
 
 
+### stamp.composers(...args)
+**(EXPERIMENTAL)**
+
+Take n functions or array(s) of functions and add the functions to the `deepConfiguration.composers` list of a new stamp. Creates new stamp. 
+* `@return {Object} stamp` The new stamp based on the original `this` stamp.
+
+Functions passed into `.composers()` are executed on the spot and every other time the new stamp or its derivatives are being composed. They are sort of a after-composition callbacks.
+
+If a composer function returns a stamp then it becomes the resulting stamp.
+
+Each function receives an options object with two properties:
+```
+{
+  composables, // The list of composables being merged
+  stamp        // The resulting stamp
+}
+```
+
+#### Examples
+
+Collect every composable the stamp was created from.
+```js
+const ComponentsMonitoring = stampit().composers(({stamp, composables}) => {
+  const conf = stamp.compose.configuration || {};
+  conf._wasComposedOf = _.uniq(composables.concat(conf._wasComposedOf));
+  stamp.compose.configuration = conf;
```

Comment:
To avoid mutations with composers a user would need to `delete stamp.compose.deepConfiguration.composers`.

> So what about just checking if something else but undefined is returned and use that instead of current stamp?

It's actually how ti works. See `stampit.js`.